index.html

```diff
@@ -30,49 +30,56 @@
-    </a>
+    <!-- BHX Pre-Registration -->
+    <div id="pre-reg">
 
-    <!-- Post-Event -->
-    <section id="post">
-        <h1>Thanks for a great BrickHack 9!</h1>
-        <p>See you next year!</p>
-        <a id="devpost" href="https://brickhack9.devpost.com/" target="_blank">VIEW PROJECTS</a>
-        <p><a href="./club.html">Check out our club</a></p>
-        <video id="prizes-ricky" autoplay muted loop playsinline>
-            <source src="./assets/bh8/videos/Prize-Ricky.mp4" type="video/mp4">
-        </video>
-    </section>
+        <h1 class = "bh-gradient gradient-text">Celebrating 10 Years</h1>
+        <h1 class = "bh-gradient gradient-text">of 
+        <span class="standard-text">BrickHack</span></h1>
 
-    <!-- Footer -->
-    <footer>
-        <div id="footer-left">
-            <p><a href="mailto:hello@brickhack.io"><b>hello@brickhack.io</b></a></p>
+        <div class="bh-gradient mini-window">
+            <div class="title-bar"><p class = "title standard-text">BrickHack X</p>
+                <div id="minimize"><img src="assets/bh10/pre-reg/Mini Windows Icons/Minimize-Button.svg"></div>
+                <div id="maximize"><img src="assets/bh10/pre-reg/Mini Windows Icons/Maximize-Button.svg"></div>
+                <div id="close"><img src="assets/bh10/pre-reg/Mini Windows Icons/Close-Button.svg"></div>
+            </div>
+            <p><span class ="desatblue-text">BrickHack X is returning in early 2024!</span><br>
+                Enter your e-mail to stay updated.</p>
+            <div id ="pre-reg-form">
+                <!-- Mailchimp Email Form reflecting from club.html -->
+            <form action="https://coderit.us11.list-manage.com/subscribe/post?u=122b09a8cef4c1f3888af8e40&amp;id=4c1af7f783" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" rel="noopener" novalidate display="none">
+                <input type="text" value="" class="required email" id="mce-EMAIL" placeholder="hacker@email.com" name="email" />
+                <input type="submit" value="SUBMIT"/>
+                <div id="mce-responses" class="clear">
+                    <div class="response" id="mce-error-response" style="display:none"></div>
+                    <div class="response" id="mce-success-response" style="display:none"></div>
+                </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+            </form>
+            </div>
         </div>
-        <div id="footer-right">
-            <p><a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf" target="_blank"><b>MLH Code of Conduct</b></a></p>
-            <p><a href="https://www.rit.edu/academicaffairs/policiesmanual/c070" target="_blank"><b>Privacy Policy</b></a></p>
-            <p>Copyright &copy; 2023 BrickHack Club</p>
+
+        <!-- Web Decorations / Assets for Backdrop -->
+
+        <div id="star-deco">
```

Comment:
For future: Let's see if we can get Design to make the backgrounds a little easier to implement